### DISCUSSION TITLE
web demo version

### DISCUSSION THREAD

**Discussion by erfanshayegani** [haotian-liu/LLaVA#273] [General]
Hi, hope all is well.

Until like 2 weeks ago, the web demo version showed "LLaVA-13B-v1-1".  But now it's "LLaVA-13B-v1-3-preview". Is there a way that I can access the "LLaVA-13B-v1-1" used in the demo somehow? I would really appreciate it.

**Comment by haotian-liu** [comment]
Hi, do you mean you want to access this in the online demo, or you want to access the model weights? If the latter, you can access the model weights [here](https://huggingface.co/liuhaotian/LLaVA-13b-delta-v1-1).

Besides, out of the curiosity, would you mind sharing the reason you want to access the v1.1 version of the demo?

**Reply by erfanshayegani** [reply]
Thanks for the reply! I'd prefer to have access to it in the online demo, but the weights should also be fine. 
Actually, I think the v1-1 version was much more creative in the responses than the new version in the demo. Even with high temperatures, I feel the new version is much more deterministic compared to the previous one. Do you have any intuitions on that?

**Reply by haotian-liu** [reply]
Hi, thanks for the feedback! I have enabled v1.1 demo again on the website. Note that this is temporary, and it may be taken down when we need GPUs (as we are just a research lab, and do not have lots of GPUs).

I would like to learn more about the creativity aspect. Can you share a few examples here on "the new version is much more deterministic compared to the previous one"? Thanks!

**Reply by erfanshayegani** [reply]
Sorry for my late reply, for example, the new version tends to stick to the same words and sentences even when I press the 'regenerate' button multiple times, this does not apply to the V1-1 version. Plus, usually, the descriptions in the new version are shorter and more general, however, the V1-1 version goes pretty detailed. I'm kind of new to this area, if it is possible for you, can you explain what has changed between these two versions?

**Reply by haotian-liu** [reply]
Hi would you provide some more concrete examples? Like the images and the prompts you used. This can let me better understand the case, and potentially further improve the checkpoints.

There are some difference in training schedules, and the base checkpoint is also different (Vicuna v1.1 vs v1.3). We’ll provide the full detail once we finalize the checkpoints.

**Reply by erfanshayegani** [reply]
Unfortunately, I don't have solid examples, the claims that I made, reflect my experience using both versions for some time. But for example, sometimes the newer version becomes much more generalistic compared to the V1.1, and does not capture as many details as possible from the image. It hallucinates more as well. If I ask it whether an object is present in the image or not, it usually says yes, however, the V1.1 version did better as far as I remember. Again sorry for my naive questions, I'm kind on new, what GPUs are required for the V1.1 version? I'm asking since you answered to me that it's temporary due to limited GPU resources in your lab.
